### ISSUE TITLE
content not included in repo

### ISSUE THREAD

**Issue by ShanEllis** [oreillymedia/ethics-datascience#1]
In the [Kindle version](https://www.amazon.com/dp/B07GTC8ZN7/ref=cm_sw_r_cp_ep_dp_fKyLBb0YWVH94), the authors state that this link will contain the books' contents will be available at this link; however, the contents are not (yet) here. Just making sure the link isn't supposed to point somewhere else?

**Comment by mikeloukides** [comment]
Yes, our intent is to publish it there. However, I haven't had time to do
it. The series is available online at
https://www.oreilly.com/tags/data-ethics.

Mike Loukides
VP, Content Strategy
O'Reilly Media, Inc.

**Comment by jpborsi** [comment]
Repo still empty. Is there a date for when we can expect it?

**Comment by jpborsi** [comment]
Thanks for the link!

**Comment by carlosvega** [comment]
I just found this other repo. I wanted to contribute to the gitlab repo in https://resources.oreilly.com/examples/0636920203964/ fixing a broken URL. The URL is the one about FB reminding people's deaths. I tried to suggest a new link with a pull request to the Gitlab repository but access is restricted. I propose to replace it with this link from archive.org  https://web.archive.org/web/20190408164600/https://www.cbc.ca/newsblogs/yourcommunity/2014/12/facebook-apologizes-for-highlighting-painful-memories-with-year-in-review-feature.html

 As a suggestion, it could be great to be able to create issues on the gitlab repo.

**Comment by mikeloukides** [comment]
Thanks. I'll get this fixed.

Mike Loukides
VP, Content Strategy
O'Reilly Media, Inc.
